### PR TITLE
[IMPROVE] User mentions

### PR DESCRIPTION
**Pull request by vynmera**
Related to #10968

It makes the mentions look nicer, and stand out from normal links (otherwise, one could change their name to an URL and be invisible)
Not sure whether it's a [FIX] or a [NEW], but it's small so I'll pick the former.

**Old**

**New**

### CODE REVIEW

File: packages/rocketchat-theme/server/colors.less [hunk 1]
```diff
@@ -598,6 +598,10 @@ input:-webkit-autofill {
 	}
 
 	.mention-link {
+		&:not(.mention-link-me):not(.mention-link-all) {
```
[the remark this answers is not in the document]
Comment (vynmera) [reply packages/rocketchat-theme/server/colors.less]: Just tried it, doesn't work :(

**Comment by yahesh** [comment]
Wow, the default now puts light-green text on a light-blue background. Why would anyone think that is an improvement to what it has looked like before?

We fixed that by using this custom CSS for now, but please fix this bad default...

```css
.message .mention-link {
  background-color: @rc-color-button-primary;
  color           : #fff;
}

.message .mention-link:hover {
  color: #fff;
}```

**Comment by vynmera** [comment]
@yahesh Do you have any custom CSS or colors? It seems to look OK on open.rocket.chat...?

**Comment by yahesh** [comment]
@vynmera We use colour-blindness improved status colours, but as far as I can see, none of these should modify `@rc-color-button-primary` that is used for background-color or `@link-font-color` that seems to be used for color:

```css

/* Improve colours for colour-blindness */
:root {
  --status-away:              #6db6ff !important;
  --status-busy:              #FF0000 !important;
  --status-invisible:         #004949 !important;
  --status-invisible-sidebar: #004949 !important;
  --status-offline:           #004949 !important;
  --status-online:            #24ff24 !important;
}
```

**Comment by yahesh** [comment]
We've looked into this some more and as it seems, the corresponding configuration has moved to "Old Colors (minor)" some time ago and is difficult to follow as `@link-font-color` is set as an expression down there.

**Comment by yahesh** [comment]
We've reverted to the default colour scheme (except for the status colours) to solve this on our side.

**Pull request merged**